ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -826,6 +826,28 @@ tests:
       enable:
       - observers-resource-watch
     workflow: telco5g-hcp-cnf
+- as: network-flow-matrix-bm
```

Comment:
These should follow our naming scheme for CI jobs

https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

Mostly I think you're missing platform and CNI provider.